### ISSUE TITLE
how to create train_label_pair file

### ISSUE THREAD

**Issue by furkanese** [guyuchao/IPCGANs-Pytorch#5]
As I understand the purpose of the file is to create fake labels of training data for the discriminator but I could not find the code to create the aforementioned file. Could you point me the direction to create it?
